### ISSUE TITLE
[ANDROID] App crash on when you select any link within Purposes/Features or Site vendors tabs

### ISSUE THREAD

**Issue by wezley98** [SourcePointUSA/android-cmp-app#197]
### Steps to Reproduce
- Select Privacy options from menu
- Expand "Purposes" heading
- Select "Google ad Manager"

### Expected
Google Ad Manager will open

### Actual
App Crashes 

App crash on when you select any link within Purposes/Features or Site vendors tabs

Tested Device:
||Samsung Galaxy S9|10.0

```
2020-07-07 11:01:18.891 7465-7465/com.bskyb.skynews.android E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.REMOVED.REMOVED.android, PID: 7465
    android.util.AndroidRuntimeException: Calling startActivity() from outside of an Activity  context requires the FLAG_ACTIVITY_NEW_TASK flag. Is this really what you want?
        at android.app.ContextImpl.startActivity(ContextImpl.java:952)
        at android.app.ContextImpl.startActivity(ContextImpl.java:928)
        at f.n.a.r.k(ConsentWebView.java:224)
        at f.n.a.r.b(ConsentWebView.java:39)
        at f.n.a.r$a.shouldOverrideUrlLoading(ConsentWebView.java:154)
        at android.webkit.WebViewClient.shouldOverrideUrlLoading(WebViewClient.java:83)
        at org.chromium.android_webview.AwContentsClientBridge.shouldOverrideUrlLoading(chromium-TrichromeWebViewGoogle.apk-stable-410410681:16)
        at android.os.MessageQueue.nativePollOnce(Native Method)
        at android.os.MessageQueue.next(MessageQueue.java:336)
        at android.os.Looper.loop(Looper.java:174)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
```

**Comment by wezley98** [comment]
This happens for any link inside `options` which tries to open webview.

**Comment by leodoin** [comment]
@wezley98 , thanks for bringing this. This fix is already merged [#191] and should be released soon.
